src/doc_builder/commands/build.py

```diff
@@ -62,7 +62,7 @@ def build_command(args):
                 "the doc-builder package installed, so you need to run the command from inside the doc-builder repo."
             )
 
-    if args.not_python_module:
+    if args.not_python_module and args.version is None:
```

Comment:
I found that we need to check that `args.version` isn't provided because the PR build uses this flag to define the subfolders. Without this change, the docs are always built to `course/main/` and we get a 

```
mv: cannot stat '../build_dir/course/pr_35': No such file or directory
```

error when we try to push to the `doc-build-dev` repo. Running the course CI again to double-check

Reply:
oh indeed :) thanks for catching this